src/commands/add/spec/add-credential.spec.ts

```diff
@@ -21,7 +21,7 @@ describe('addCredential', () => {
 
     expect(inputModule.getAndValidateServerUrl).toHaveBeenCalled()
     expect(configUtils.saveToLocalConfig).toHaveBeenCalledWith(
-      new Target({
+      jasmine.objectContaining({
```

Comment:
why do we need `jasmine` here?

Reply:
`Target` has all the attributes with default values, to partial check with concerned attributes.

https://jasmine.github.io/2.0/introduction#section-Partial_Matching_with_%3Ccode%3Ejasmine.objectContaining%3C/code%3E

Reply:
This is the `jest` way of doing it: https://jestjs.io/docs/en/expect#expectobjectcontainingobject